src/main/java/com/neprozorro/rest/mockdata/ResultReportsMockDataGenerator.java

```diff
@@ -1,28 +1,32 @@
-package com.neprozorro.rest;
+package com.neprozorro.rest.mockdata;
 
-import com.neprozorro.service.MockService;
+import com.neprozorro.service.mockdata.ReportsMockDataGeneratorService;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
 @Slf4j
 @RestController
 @RequiredArgsConstructor
-public class MockResource {
+@RequestMapping("/mockdata/result-report-generator")
+public class ResultReportsMockDataGenerator {
 
-    private final MockService saveService;
+    private final ReportsMockDataGeneratorService mockService;
 
-    @GetMapping("/mockdata")
+    @GetMapping
     @ResponseStatus(HttpStatus.FOUND)
-    public void save() {
+    public void generateResultReportsMockData() {
```

Comment:
if it's Generate we create resources, shouldn't it be POST?

Reply:
Done